package.json

```diff
@@ -104,7 +104,7 @@
     "webpack-addons": "^1.1.5",
     "yargs": "^11.0.0",
     "yeoman-environment": "^2.0.0",
-    "yeoman-generator": "github:ev1stensberg/generator#Feature-getArgument"
+    "yeoman-generator": "^2.0.3"
```

Comment:
Nice!

Reply:
Most awaited. :sweat_smile:

Reply:
Also, update the PR description to include "Fixes #161" :rocket: